### ISSUE TITLE
Fix SIP Go Code Style Issues

### ISSUE THREAD

**Issue by lb4368** [att-comdev/sip#18]
Need to address linter warnings identified with code comments, variable names, etc. and remove commented lines of code in SIP POC code, so that code/comment style matches best practice.

**Comment by ian-howell** [comment]
I don't know that this has a clear closing criteria since code stewarding is an ongoing process, but I'll go ahead and take some time to clean things up.

**Comment by teoyaomiqui** [comment]
> I don't know that this has a clear closing criteria since code stewarding is an ongoing process, but I'll go ahead and take some time to clean things up.

i would say closing criteria is to add a target in make file that would run reasonable lint tests, and code passes them

**Comment by lb4368** [comment]
> 
> 
> > I don't know that this has a clear closing criteria since code stewarding is an ongoing process, but I'll go ahead and take some time to clean things up.
> 
> i would say closing criteria is to add a target in make file that would run reasonable lint tests, and code passes them

Yes, closing criteria for this is for the current version of code to pass lint tests.  It should be an on-going acceptance criteria for new features.  There is also a https://itrack.web.att.com/browse/CPVYGR-94 story in iTrack to add lint to the build.

**Comment by ian-howell** [comment]
Closing criteria will be met via #21, although I am going to continue fixing some style/spelling/formatting issues
